### DISCUSSION TITLE
Can't get direct solution of odes when solving inverse problem

### DISCUSSION THREAD

**Discussion by ZSTanone** [lululxvi/deepxde#992] [General]
Dear all,
    I recently used the PINN library to solve the inverse problems of odes，and I successfully made the parameter estimation. 
However，in such simple example you can get your observation as the solution of your odes easily. As for something different ,which you can not get the direct solution of your dynamic equations，but combinations of solutions.

   For example ，there are some odes and I use **_di（i = 1,2,3...）_** to denote them . In easy problems, I may have the numerical solutions of di（not all）,and I can set them as observation points directly. In complex problems , I may have **_C1 = (d1 + d2)/(d3 + d4) , C2 = (d5 + d6) ......_** ，C1 and C2 are the values that I can measure directly，which can not use for observation points. However, before doing parameters estimation，I already done the structurally identifiability of my model. 
    
How can I apply PINN to solve inverse problem that use the indirect measurements. Is that possible to change the structure of normal PINN？ or you can provide me more ideas

**Comment by lululxvi** [comment]
Many ways to do it. The easy is to use `PointSetOperatorBC` from https://github.com/lululxvi/deepxde/pull/986. Will be merged soon.
